pnt-lut.c

```diff
@@ -132,6 +143,14 @@ int main(int ac, char *av[]) {
       syslog(LOG_DEBUG, "output is 0x%x", output[1]);
 #endif
       write(fd_out, output, 2);
+      
+      // write image to file if SIGINT is detected
+      if (sig_rcvd) {
+        const int fd1 = open("image.txt", O_WRONLY);
+        write(fd1, global, PGM_SIZE);
+        close(fd1);
```

Comment:
I was thinking this should be more like:
```
  const char *pgm_init = "P5 " STR(PGM_W) " " STR(PGM_H) " " STR(PGM_DEPTH) " ";
  write(pptr, pgm_init, strlen(pgm_init));
  for (unsigned jj = 0; jj < PGM_H; jj++) {
    for (unsigned ii = 0; ii < PGM_W; ii++) {
      write(pptr, &pgm_buf[ii][jj], 1);
    }
  }
  syslog(LOG_INFO, "wrote PGM size %zu\n", sizeof(pgm_buf));
  close(pptr);
  ```
Because this actually makes a PGM file instead of just a text file of bytes